package.json

```diff
@@ -58,6 +58,7 @@
 		"*": "prettier --ignore-unknown --write"
 	},
 	"dependencies": {
+		"@altano/repository-tools": "^0.1.1",
```

Comment:
> Performance: the library can find a git repository's root in ~3ms, and the worst (slowest) repository type (Mercurial) takes ~148ms. I think this is totally acceptable for package.json linting:

Agreed. I have yet to meet someone who uses Mercurial day-to-day and is thrilled enough about it to want to optimize that seventh of a second.